src/flp/types/fixedpoint_l2.rs

```diff
@@ -316,13 +343,21 @@ where
     F: FftFriendlyFieldElement,
     SPoly: ParallelSumGadget<F, PolyEval<F>> + Eq + Clone + 'static,
     SBlindPoly: ParallelSumGadget<F, BlindPolyEval<F>> + Eq + Clone + 'static,
+    F::Integer: TryFrom<u128>,
```

Comment:
The trait bounds here are already complex, I'd hate to make them more so! It might be worthwhile to be concrete for `FixedPointBoundedL2VecSum`. Do you imagine ever needing a field larger than `Field128`? I don't think we can safely use a smaller field, so if not I'd suggest replacing the generic `F` with `Field128` and simplify the bounds.